torch/nn/modules/loss.py

```diff
@@ -1235,24 +1240,45 @@ class CTCLoss(_Loss):
 
     Inputs:
         log_probs: Tensor of size :math:`(T, N, C)` where `C = number of characters in alphabet including blank`,
-            `T = input length`, and `N = batch size`.
+        `T = input length`, and `N = batch size`.
             The logarithmized probabilities of the outputs
             (e.g. obtained with :func:`torch.nn.functional.log_softmax`).
-        targets: Tensor of size :math:`(N, S)` or `(sum(target_lengths))`.
-            Targets (cannot be blank). In the second form, the targets are assumed to be concatenated.
+        targets: Tensor of size :math:`(N, S)` or `(sum(target_lengths))`, where `N = batch size`, and
+        `S = maximum target length`.
+            Each element in the target sequence is a class index. Target index cannot be blank (default=0).
+            In the second form, the targets are assumed to be concatenated.
```

Comment:
Are the targets concatenated with or without the padding to make it a fixed (N, S)?
Maybe it would be useful to explicitly comment on that.

Reply:
I think the answer is "with padding". I actually still don't understand how the (sum(target_lengths)) form works... it doesn't seem like that actually describes a shape. Can @t-vi comment on this?

Reply:
That's a 1-d shape. `(sum(target_lengths),)` would probably be more pythonesque.

Reply:
Ah no I see now. Not sure why I was confused before. The comma is probably not necessary.